### PR TITLE
leecalcote/blog/v08 announcement

### PR DESCRIPTION
**Pull request by leecalcote**
- **chore: upgrade-ruby-versions**
- **[Catalog] Update pagination settings and add Meshery v0.8.0 release announcement**

### CODE REVIEW

**Review by Jougan-0 — DISMISSED** [review]

File: collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+* **End-to-End Testing:**  Increased focus on end-to-end testing with a goal of 50% coverage.
+* **Playwright Adoption:**  Transitioning from Cypress to Playwright for more robust testing.
+* **Automated Test Results:** Stay informed with automatically published end-to-end test results.
+
+### Join the Meshery Community!
+
+We invite you to explore Meshery v0.8.0 and experience the future of cloud native management.  Join us on [Slack](https://slack.meshery.io/) to connect with the community, share your feedback, and contribute to the project.
+
+New Maintainer: [Sangram Rath](https://layer5.io/community/sangram-rath)
```
Comment (sangramrath) [inline collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md:93]: I guess this should be https://layer5.io/community/members/sangram-rath

Thank You for the mention here.

File: collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+A big thank you to all of the contributors who made this release possible! We are grateful for your continued support of the Meshery project.
+
+## Looking Ahead and onto v0.9
+
+While v0.8.0 delivers significant advancements, we're already working on v0.9.0! 
+
+<!-- Checkout [Meshery’s roadmap](https://github.com/meshery/meshery/blob/master/ROADMAP.md) to peek into what’s yet to come. -->
+
+We are already hard at work on [v0.9.0 Roadmap](https://discuss.layer5.io/t/meshery-v0-9-roadmap/6296)! See [v0.9.0 Milestone](https://github.com/meshery/meshery/milestone/5) in `meshery/meshery` for more details and the [Meshery Architecture deck](https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#) for additional visual designs and functional specifications. Stay tuned for more updates.
```
Comment (sangramrath) [inline collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md:111]: The milestone link should be https://github.com/meshery/meshery/milestone/6, isn't it?

File: collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+<ul>
+<li>Multiple Telemetry Providers and Custom Boards (<a href="https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#slide=id.g1044af767ce_5_2">spec</a>)</li>
+<li>User-defined Dashboards and Metrics (<a href="https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#slide=id.gcb74201a11_0_119">spec</a>)</li>
+</ul>
+  
+</details>
+
+**Dive deeper into the details and explore the full list of changes in the [Meshery Documentation](https://docs.meshery.io/docs/project/releases). Stay tuned for updates on [v0.9.0](https://discuss.layer5.io/t/meshery-v0-9-roadmap/6296)!**
+
+Try v0.8.0 in the [Meshery Playground](https://play.meshery.io) or [download](https://docs.meshery.io/installation) the latest release to get started. We look forward to your feedback and contributions!
```
Comment (sangramrath) [inline collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md:151]: Should Meshery Playground link be https://playground.meshery.io or https://kanvas.new? Or it is deliberately at https://play.meshery.io for the onboarding flow?

File: collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md (hunk 1 above)
Comment (leecalcote) [reply collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md:93]: Good catch. Yes.

I'll add a little more context to this mention, too.

File: collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md (hunk 2 above)
Comment (leecalcote) [reply collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md:111]: Oh, my goodness. Yes, thank you. Updated.

File: collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md (hunk 3 above)
Comment (leecalcote) [reply collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md:151]: Great question. Yes, you're right that it is listed as play.meshery.io for the overview and explantation of what it is, why you need to sign in, and so on prior to pointing users directly to a sign-up/sign-in page.

**Review by miacycle — APPROVED** [review]
Very cool!

**Review by jamieplu — APPROVED** [review]
I'm so proud!

**Comment by winkletinkle** [comment]
This is HUGE

**Pull request merged**